.github/workflows/push.yml

```diff
@@ -612,7 +618,7 @@ jobs:
           password: ${{ secrets.SERVICE_ACR_PASSWORD }}
       - name: Push image to ACR
         env:
-          VERSION: ${{ needs.set-version.outputs.version }}
+          VERSION: ${{ steps.strip_prefix.outputs.version }}
```

Comment:
@JakubMatejka FYI najlos asi rata, ze aj image tag zacina s `templates-api-v*`, mali by ste sa syncnut:
https://keboola.atlassian.net/wiki/spaces/KB/pages/2512551997/Templates+API+draft

+ ty tu nemas to `-v`, takze image tagy budu `v1.2.3`, za mna OK, len aby to sedelo s pipeline v Azure.

Reply:
Aha, už to čtu. @ondrejhlavacek Takže release pipeline počítá s tagem např. `templates-api-v1.0.0` a prefix stripovat nemám. Tak to ještě fixnu.

Reply:
Jo jasně `v` mi tam chybí, už to chápu. 😅